### PR TITLE
Authentication for UI draft

### PR DESCRIPTION
**Pull request by Woselko**

### CODE REVIEW

File: Actively/Actively/ActivelyDomain/Entities/User.cs
Comment (EmilBulka) [inline Actively/Actively/ActivelyDomain/Entities/User.cs:1]: please change Gender for enum. Also remove attributes to keep property configurations in one class "User Configuration"

File: Actively/Actively/ActivelyTests/ActivelyApp.Tests/ControllersTests/AuthenticationControllerTests.cs
Comment (EmilBulka) [inline Actively/Actively/ActivelyTests/ActivelyApp.Tests/ControllersTests/AuthenticationControllerTests.cs:1]: why using is commented? if it's unused and unnecessary, remove it

**Review by EmilBulka — CHANGES_REQUESTED** [review]
please read comments and do small changes

File: Actively/Actively/ActivelyDomain/Entities/User.cs
Comment (Woselko) [reply Actively/Actively/ActivelyDomain/Entities/User.cs:1]: resolved in last commit

File: Actively/Actively/ActivelyTests/ActivelyApp.Tests/ControllersTests/AuthenticationControllerTests.cs
Comment (Woselko) [reply Actively/Actively/ActivelyTests/ActivelyApp.Tests/ControllersTests/AuthenticationControllerTests.cs:1]: removed, thanks

**Comment by Woselko** [comment]
> please read comments and do small changes

Done, thanks

**Review by EmilBulka — APPROVED** [review]

**Pull request merged**